src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -464,6 +533,47 @@ def register_submission(self, submission: Submission, options: dict) -> dict | N
                 "intermediate.objects_api_zaakobject",
             )
 
+        # Connect variables with zaak eigenshappen (if a connection has been used in ZGW registration)
+        if variables_properties := options.get("variables_properties"):
+            mapped_variables_properties = get_variables_properties_from_submission(
+                submission, variables_properties
+            )
+
+            eigenshappen = execute_unless_result_exists(
+                partial(catalogi_client.list_eigenshappen, options["zaaktype"]),
+                submission,
+                "intermediate.eigenshap",
```

Comment:
```suggestion
                "intermediate.zaaktype_eigenschappen",
```
probably better to namespace this a bit to improve readability